src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -552,8 +552,21 @@ class SnykToolWindowPanel(
 
         val newContainerTreeNodeText = getNewContainerTreeNodeText(settings, containerResultsCount, addHMLPostfix)
         newContainerTreeNodeText?.let { rootContainerIssuesTreeNode.userObject = it }
+
+        val newRootTreeNodeText = getNewRootTreeNodeText()
+        newRootTreeNodeText.let { rootTreeNode.info = it }
     }
 
+    private fun getNewRootTreeNodeText() : String {
+        val folderConfig = service<FolderConfigSettings>().getFolderConfig(project.basePath.toString())
```

Comment:
You shouldn't use the `project.basePath` as stated in the IntelliJ SDK docs. In the future we'll most likely use content root nodes like in Eclipse in the future. We can pair on this to explore how to solve this issue, but I feel like using `project.basePath` with folderConfigs creates inconsistencies.

Reply:
That said, it's like that in the other code, too. Just wondering if we can improve it.

Reply:
Ok, as discussed this is part of a bigger issue, let's pair on this sometime and figure out all the instances of the `project.basePath` that we use where we should use the content root.